src/main/java/uk/gov/ida/eidas/cli/metadata/SignWithSmartcard.java

```diff
@@ -25,7 +26,7 @@
 
   @Override
   public Void call() throws Exception {
-    PKCS11KeyLoader keyLoader = new PKCS11KeyLoader(sun.security.pkcs11.SunPKCS11.class, pkcs11Config, password);
+    PKCS11KeyLoader keyLoader = new PKCS11KeyLoader(Security.getProvider("SunPKCS11").getClass(), pkcs11Config, password);
```

Comment:
Do we need any other additional config for this to work?

Reply:
you can test it out with the local jar to see it work.

@tomrosier nice avatar 👌

Reply:
It just seems to wrap `sun.security.pkcs11.SunPKCS11` and make it accessible through the 'standard Java api'. Will local test it now 😄

Reply:
Also Cheers Mino! @minhngocd